lib/utils/opType.js

```diff
@@ -47,4 +47,12 @@ module.exports = {
 			logString: withOpType[2],
 		};
 	},
+	opTypes: {
+		assertLine: 'A',
```

Comment:
you have introduced this constant but it is not used anywhere

Reply:
I made it explicitly for other developers for presenting which operation types currently used in library.

Reply:
A and E are already used somewhere, I would expect this PR to refactor those usages into usages of this introduced constant.